### PR TITLE
Create 2.7.0-rc2 version

### PR DESCRIPTION
**Pull request by zenmonkeykstop**
## Status

Ready for review

## Description of Changes

Towards #6979.

Changes proposed in this pull request:

## Testing

- [x] changelog update is consistent
- [ ] CI is passing
- [x] PR base is release/2.7.0
- [x] version updated correctly.

### CODE REVIEW

File: changelog.md
Comment (cfm) [inline changelog.md:7]: Optional nit (original #7008 → backport #7010):

```suggestion
* Update CI logic for continuous localization (#7010)
```

File: changelog.md
Comment (cfm) [inline changelog.md:11]: Optional nit: original #7009 → backport #7015.

**Review by cfm — APPROVED** [review]
Approving pending CI.  Nits are optional; I won't be offended if you don't feel like restarting the CI clock for them.  :-)

File: changelog.md
Comment (zenmonkeykstop) [reply changelog.md:7]: Since we're grouping all those in the final changelog, I'm just using the same text for now while making it clear which pr went in which release.

File: changelog.md
Comment (zenmonkeykstop) [reply changelog.md:11]: In general IMO it's more useful to reference the original PR rather than the backport.

Comment (cfm) [reply changelog.md:11]: I agree; that's why I flagged <https://github.com/freedomofpress/securedrop/pull/7018#discussion_r1366209819>.

Comment (zenmonkeykstop) [reply changelog.md:11]: d'oh - we'll fix it in post :movie_camera:

**Review by cfm — APPROVED** [review]
Troubleshot out of band:  `app-tests` and `app-page-layout-tests` are passing, but `uploader.codecov.io` is flapping between previous version 0.7.0 and new version 0.7.1.  I'm comfortable approving this as it is; your merge as Release Manager, @zenmonkeykstop.

**Pull request merged**
